test/json.go

```diff
@@ -0,0 +1,24 @@
+package test
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+)
+
+// ReadJSON is a testing utility that reads data from JSON file into an
+// interface and fails with FailNow on error.
+//
+//   cfg := Config{}
+//   test.ReadJSON(t, "testdata/config.json", &cfg)
+func ReadJSON(t testing.TB, fname string, v interface{}) {
+	t.Helper()
+	f, err := os.Open(fname) // nolint:gosec
```

Comment:
no space after comment marker for tool directives.

Reply:
done.